js/api/Surveys.js

```diff
@@ -216,16 +226,17 @@ export function loadSurveys(callback) {
 /**
  * fetches remote data for surveys and invitations
  *
+ * @param {object}   options, options object to be passed to loadSurveys
  * @param {function} done, the callback for when the async operations are done
  */
-export function loadSurveyList(done) {
+export function loadSurveyList(options = {}, done) {
   async.auto({
-    surveys: (cb) => {
-      loadSurveys(cb);
-    },
     invitations: (cb) => {
       loadInvitations(cb);
     },
+    surveys: ['invitations', (cb) => {
```

Comment:
Originally we cached every single Form,Question,Trigger available from the Parse server straight into the user's device, including pending and declined surveys. V1's UI gets very sluggish for this reason.

The request chain is not a perfect solution, and of course I'm always looking for ideas on how to better handle this.